### DISCUSSION TITLE
Recently I started getting this in my app's logs - WARNING:nextcord.state:nextcord.Client: Forbidden error

### DISCUSSION THREAD

**Discussion by epsilonion-liam** [nextcord/nextcord#1233] [Q&A]
Recently I started getting this in my app's logs, they must of changed something as it was fine before.

WARNING:nextcord.state:nextcord.Client: Forbidden error for 1297945774780317696, is the applications.commands Oauth scope enabled? 403 Forbidden (error code: 50001): Missing Access

All scopes and permissions are set fine and all the cog's seem to be working fine so don't know if its a false positive or not?

**Comment by Nuwan-Srimal** [comment]
It’s not a harmless false positive, it means your bot wasn’t invited with the applications.commands scope (or is missing access to sync commands in at least one guild). Re-inviting with the correct scope should clear it.
